deploy/k8s/overlays/prod/kustomization.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: labelstudio
+
+namePrefix: labelstudio-
+
+bases:
```

Comment:
`bases` field was deprecated in kustomize v2.1.0. Move entries into the resources field

Reply:
will do!